### ISSUE TITLE
Code for CIFAR experiments

### ISSUE THREAD

**Issue by BenCrulis** [trzhang0116/feature-contamination#1]
Hello, are you planning to add the code for the experiments on CIFAR that appear in Figure 6 in the paper?

**Comment by trzhang0116** [comment]
Hi Ben, we may release those codes later in this repo after some clean-up.

**Comment by trzhang0116** [comment]
Hi, we have added the code for CIFAR-10 experiments. You can check the `cifar/` folder for more details!

**Comment by BenCrulis** [comment]
Hi @trzhang0116 , thank you for providing the code and the readme! I will be sure to try my method on CIFAR too then.

And by the way, I suspected that feature contamination had something to do with grokking too, but my method does not seem to help the model grok when I tried in Neel Nanda's notebook https://www.neelnanda.io/blog/interlude-a-mechanistic-interpretability-analysis-of-grokking
It might also just be that I need to tune the hyperparameters better or that I implemented something incorrectly.

Also, I recently found this preprint that might also be relevant to feature contamination https://arxiv.org/abs/2406.16935
